### PR TITLE
add FDLIBM license to openlibm ebuild

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Forgot to add FDLIBM license
see
https://bugs.gentoo.org/show_bug.cgi?id=614928
https://github.com/JuliaLang/openlibm/blob/master/LICENSE.md
https://lists.fedoraproject.org/pipermail/legal/2013-December/002343.html

### CODE REVIEW

File: licenses/FDLIBM [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+
+ Developed at SunPro, a Sun Microsystems, Inc. business.
+ Permission to use, copy, modify, and distribute this
+ software is freely granted, provided that this notice
+ is preserved.
```
Comment (mgorny) [inline licenses/FDLIBM]: Can't we match that to one of the existing licenses? @gentoo/licenses?

**Comment by SoapGentoo** [comment]
@Alessandro-Barbieri:
**rebase**, not **merge**. Learn how `git rebase` is the Swiss army knife for modifying git branches:

- https://wiki.gentoo.org/wiki/Gentoo_GitHub#Rebasing_commits
- http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

1. You want to `git pull --rebase` from the upstream branch to get rid of the merge commit you created
2. Please follow our **COMMITMSG** guidelines (https://devmanual.gentoo.org/ebuild-maintenance/index.html#git-commit-message-format). Please also consider using repoman for committing, as it autoformats the **COMMITMSG**s.

File: licenses/FDLIBM (hunk 1 above)
Comment (Alessandro-Barbieri) [inline licenses/FDLIBM]: On the Fedora mailing list they said
> Let's just call that SunPro license MIT and move on. It isn't unique
> enough to merit a new license name.

**Comment by ulm** [comment]
>>>>> On Sun, 23 Apr 2017, Alessandro Barbieri wrote:

> On the Fedora mailing list they said

>     Let's just call that SunPro license MIT and move on. It isn't
>     unique enough to merit a new license name.

Let's do the same (as we have done before). We can call it either MIT
or ISC, both are very close to it.

**Comment by mgorny** [comment]
Ok, so looks like no change is needed.

**Pull request closed without merging**